pytripgui/app_logic/app_callbacks.py

```diff
@@ -39,14 +40,14 @@ def add_empty_patient(self):
             self.app_model.patient_tree.add_new_item(None, patient)
 
     def on_open_voxelplan(self):
-        patient = PatientItem()
-        if self.open_voxelplan_callback(patient):
-            self.app_model.patient_tree.add_new_item(None, patient)
+        path = self.parent_gui.browse_file_path("Open Voxelpan", "Voxelplan (*.hed)")
+        logger.debug("Open Voxelplan: " + path)
```

Comment:
In logger it is better to use `"Open ...".format(path)` than to concatenate strings